repository/src/main/java/org/alfresco/repo/search/impl/DbOrIndexSwitchingQueryLanguage.java

```diff
@@ -110,6 +113,11 @@ public void setHybridEnabled(boolean hybridEnabled)
         this.hybridEnabled = hybridEnabled;
     }
 
+    public void setSubsystemName(String subsystemName)
+    {
+        this.subsystemName = subsystemName;

```

Comment:
Note: only used for logging. I'm sure if there is a way to derive at runtime (instead of config property).